OpenRA.Game/Network/UPnP.cs

```diff
@@ -25,7 +25,8 @@ public static void TryNatDiscovery()
 				NatUtility.Logger = Log.Channels["server"].Writer;
 				NatUtility.Verbose = Game.Settings.Server.VerboseNatDiscovery;
 				NatUtility.DeviceFound += DeviceFound;
-				NatUtility.DeviceLost += DeviceLost;
+				// Mono.Nat never raises the DeviceLost event
```

Comment:
I managed to get it working myself using https://www.nuget.org/packages/AsyncBridge/. See https://github.com/OpenRA/OpenRA/pull/7692